sdk/servicebus/Microsoft.Azure.ServiceBus/tests/Primitives/ConcurrentExpiringSetTests.cs

```diff
@@ -27,5 +27,27 @@ public void Contains_returns_false_for_expired_entry()
             set.AddOrUpdate("testKey", DateTime.UtcNow - TimeSpan.FromSeconds(5));
             Assert.False(set.Contains("testKey"), "The set should return false for an expired entry.");
         }
+
+        [Fact]
```

Comment:
Could you add tests where there are more than 2 elements in the collection?

Reply:
Sure